LICENSE

```diff
@@ -1,17 +1,15 @@
-#######
-License
-#######
+check_po, python package to automate task on traduction file .po
+Copyright (C) 2016  aRkadeFR
```

Comment:
Whatever the license was before, I'm not really comfortable with this copyright update:

```diff
+Copyright (C) 2016  aRkadeFR
-Copyright (C) 2012 Rémy HUBSCHER <remy.hubscher@novapost.fr>
```